Dockerfile

```diff
@@ -13,20 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM openjdk:8
+FROM openjdk:8-alpine3.9
```

Comment:
I don't think I spotted it to be fair, it's about 3M smaller so might be worth it and as you say it seems to be getting kept up to date.

Note, all values here unless specified otherwise are from the Docker command.

I did a quick rebuild with the new one and this is the sizes:

- openjdk:8-alpine3.9   1058822286 bytes
- openjdk:8-jre-slim      1136341123 bytes (+73 MB)

So it was actually larger to use the smaller base image, not what I expected, I did have to add wget and removed the installation of bash from the current Dockerfile as it was already present in 8-jre-slim

On disk the pulled images were different to dockerhub values
- openjdk:8-alpine3.9 - 105MB (disk) 70.09MB (dockerhub)
- opendjk:8-jre-slim - 184MB (disk) (dockerhub)

Looking into it, it seems that dockerhub reports the compressed size, not the size on disk once you pull the image. 

I did the comparison for the final image sizes:
- openjdk:8-alpine3.9 base - 1.06GB
- openjdk:8-jre-slim base - 1.14GB

I tested both builds so they both function the same, perhaps taking the 80MB increase is worth it for being up to date?

What are your thoughts, stick with what we have or update to 8-jre-slim?

Reply:
I am in favor of 8-jre-slim because its more up to date and its based on Debian which I think a larger base of contributors and users may be familiar with.

Reply:
Very nice analysis and write up of the analysis.